Makefile

```diff
@@ -281,3 +281,17 @@ genproto_sub:
 # to proto and after running `make genproto`
 genpdata:
 	go run cmd/pdatagen/main.go
+
+# The following commands can only be run on Windows (use WSL2 or Cygwin,
+# or run equivalent commands directly in Cmd/Powershell)
```

Comment:
Not sure if its worth adding this? It's very useful to have a command that you can run locally on Windows to build an msi, especially to support the first bullet point in https://github.com/open-telemetry/opentelemetry-collector/issues/1125. However, the Wix Toolset only works on Windows, so these commands will not work on Linux/Mac. And make commands are not the best way to define this on Windows either (although if WSL2 goes GA soon then maybe this is okay).

Side note: I experimented with [wixl](https://wiki.gnome.org/msitools) for quite a while but it doesn't support all wix components, such as CopyFile or most CustomAction elements, and wasn't worth the effort in the end.

Reply:
@james-bebbington Yes, I suggest moving this to a separate powershell script, something like `scripts/msi/make.ps1`.  Then we can run it as `.\scripts\msi\make.ps1 -version VERSION -config CONFIG`.